Homework_Basic_Python.ipynb

```diff
@@ -0,0 +1,209 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Assignment 1\n",
+    "\n",
+    "Calculate the sum of all the squared numbers from 1 to 100. Keep in mind that the `range()` function excludes the given upper limit."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "385\n"
+     ]
+    }
+   ],
+   "source": [
+    "uno = [d**2 for d in range(1,11)]\n",
+    "#print(uno)\n",
+    "print(sum(uno))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Assignment 2\n",
+    "\n",
+    "Calculate the sum of all the numbers in an array. There are multiple ways of doing this, choose any you like."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Length of array: 4\n",
+      "enter number 4\n",
+      "enter number 4\n",
+      "enter number 4\n",
+      "enter number 4\n",
+      "Sum of numbers is this array is 16\n"
+     ]
+    }
+   ],
+   "source": [
+    "dos = []\n",
+    "num = int(input('Length of array: '))\n",
+    "for n in range(num):\n",
+    "        numbers=(int(input('enter number ')))\n",
+    "        dos.append(numbers)\n",
+    "print('Sum of numbers is this array is', sum(dos))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Assignment 3\n",
+    "\n",
+    "Adding to assignment 2, implement the same functionality but allow the array to be a mix of numbers and numbers represented as strings, e.g. [1, 5, '7', 5, '10']."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 45,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Length of array: 3\n",
+      "enter number or string 3\n",
+      "enter number or string '4'\n",
+      "enter number or string 4\n",
+      "['3', \"'4'\", '4']\n"
+     ]
+    }
+   ],
+   "source": [
+    "import numpy as np\n",
+    "tres=[]\n",
+    "num = int(input('Length of array: '))\n",
+    "for n in range(num):\n",
+    "        numbers=(input('enter number or string '))\n",
```

Comment:
this will make all values in the list a string, but it should be a mix. You don't have to ask the user here, just create an array with a mix of numbers and numbers as strings.